doc/api/deprecations.md

```diff
@@ -2467,6 +2467,22 @@ The legacy HTTP parser, used by default in versions of Node.js prior to 12.0.0,
 is deprecated. This deprecation applies to users of the
 [`--http-parser=legacy`][] command-line flag.
 
+<a id="DEP0XXX"></a>
+### DEP0XXX: `assert.throws/rejects` validation functions not returning `true`
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: Runtime deprecation.
+-->
+
+Type: Runtime
+
+Returning anything besides `true` as result of an `assert.throws()` or
+`assert.rejects()` validation function is deprecated. Doing so results in the
+caught error from being thrown again. That is likely not desirable. Please throw
```

Comment:
This presumably is either:

"Doing so prevents the caught error from being thrown again."

Or:

"Doing so results in the caught error being thrown again."

Whichever one is correct, can you update the wording as appropriate?